ts/features/messages/components/MessageDetail/MessageDetailsHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { PropsWithChildren } from "react";
+import { StyleSheet, View } from "react-native";
+import {
+  ContentWrapper,
+  Divider,
+  H3,
+  LabelSmall,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { ServiceId } from "../../../../../definitions/backend/ServiceId";
+import { localeDateFormat } from "../../../../utils/locale";
+import I18n from "../../../../i18n";
+import { logosForService } from "../../../../utils/services";
+import { useIOSelector } from "../../../../store/hooks";
+import { serviceByIdSelector } from "../../../../store/reducers/entities/services/servicesById";
+import { OrganizationHeader } from "./OrganizationHeader";
+
+export const gap = 8;
```

Comment:
If we really want to reuse this value, I suggest we move it under the `index` file of the `utils` folder, in order not to have a component declaring a dependency upon another component just for a number.